tests/performance/test_api/test_api.py

```diff
@@ -23,7 +38,9 @@ def test_api_endpoints(test_case, test_configuration, set_api_test_environment,
     """Make an API request for each `test_case`. `test_configuration` fixture is only used to add metadata to the
     HTML report."""
     # Apply xfails
```

Comment:
After the first test run, it fails due to it does not remove created agents and groups. Every test should be independent and this issue could produce errors not only in this test but the rest of the qa suite tests.

[R2](https://github.com/wazuh/wazuh-qa/files/7076940/r2_test_api_performance.zip)  :red_circle: